mirrormanager2/lib/hostconfig.py

```diff
@@ -25,9 +25,14 @@
 
 import mirrormanager2.lib
 
+import logging
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
```

Comment:
I like the use of ```__name__``` as it gives a nice python path to this module, which is handy for quickly identifying where logging statements came from.